modules/tinymce/src/plugins/table/main/ts/selection/SelectionTargets.ts

```diff
@@ -9,9 +10,24 @@ import * as TableSelection from './TableSelection';
 
 export type SelectionTargets = ReturnType<typeof getSelectionTargets>;
 
+type UiApi = Menu.MenuItemInstanceApi | Toolbar.ToolbarButtonInstanceApi;
+
+// Not sure if the details should vary by the target e.g. there is RunOperation.onCell, onPasteable, etc.
+// TODO: Ideally would use custom extractors in TableOperations.ts
```

Comment:
Is this comment still accurate? As I believe those extractors were moved to RunOperations?

Reply:
I have reworked the file a little to use some of new functions from RunOperation. I don't think it is ideal at the moment, but is hopefully a step in the right direction.